src/App.js

```diff
@@ -5,10 +5,10 @@ function App() {
   return (
      
        <Routes>
-         <Route path="/landingpage" element={<LandingPage />} />
+         <Route path="/" element={<LandingPage />} />
```

Comment:
You can make a seprate component to store all these route..so that your app.js look clean